packages/coil-extension/src/popup/components/TipAmountFeedback/RestrictedMessage.tsx

```diff
@@ -39,7 +39,12 @@ export const RestrictedMessage = () => {
       return (
         <Typography variant='subtitle1'>
           Max tip amount.{' '}
-          <LinkUnderlined onClick={tabOpener(`${coilDomain}/settings/tipping`)}>
+          <LinkUnderlined
+            tabIndex={0}
```

Comment:
Could this LinkUnderlined thing be a component? 

eg.  `<CoilLink path='/settings/billing'>`

Reply:
Yeah I think so, I'll update.